.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,27 @@
+**Android version and Phone Model**
+
+Add the Android version and the phone model on which the issue was encountered.
```

Comment:
You can make this as HTML comment so that this doesn't appear in the issue by default. Also, add a short example of what can go here.

Follow the same for other places too (wherever applicable).